### ISSUE TITLE
Page Build Fail - error in config.yml file

### ISSUE THREAD

**Issue by VickyTEI** [VickyTEI/VisMat-Metadata-Framework#1]
Hi @chrisdaaz - the page build has failed.  Where do I find the url, I feel like this is something that needs to be set?

The actual fail message is:

> You have an error on line 3 of your _config.yml file. 

# Gems is mentioned on line 2 - is this anything to do with the issue?
lines 1 - 5 are:

1 # Build settings
2 # Gems
3 gems: 
4 - jekyll-paginate
5 - jekyll-feed

thanks

**Comment by chrisdaaz** [comment]
I submitted a PR (#2) for this specific error, but I think there might also be another one in `_config.yml`

The full URL for your site will be this: https://vickytei.github.io/VisMat-Metadata-Framework/

I think the fields in the config file should look like this:

```
url: "https://vickytei.github.io"
enforce_ssl: ""
baseurl: "/VisMat-Metadata-Framework"
```

**Comment by VickyTEI** [comment]
OK thanks @chrisdaaz , I'm learning by mistakes. I acted on the yaml formatting check and didnt compute a PR as a Pull Request... so I edited the file before opening the pull request & now have conflicts. I'll re-read the Git History tutorial

So looking at the log - what my mind would find easiest is to revert to dc9720d. I think I have reverted back to that point on GitHub desktop simply by reverting each commit - but I am confused. I will try merging your branch again. I feel like I'm missing the obvious

**Update:
So I have 3 branches but the the gh-pages one & chrisdaaz ones look like they have the same yaml code - the conflict message did highlight the extra you had added but I was unclear quite what I was doing.... advice would be amazing, thanks

In the mean time I'll plan the basics of the site - our working plans have changed some what, so I will simplify it for now and stick more to the DAMS setup
